packages/docs/content/components/footer/healthcare-footer.mdx

```diff
@@ -0,0 +1,29 @@
+---
+title: Healthcare.gov Footer
+healthcare:
+  githubLink: ds-healthcare-gov/src/components/Footer
+  sketchLink: 52ykDvL
+---
+
+import { Alert } from '@cmsgov/design-system';
+
+<ThemeContent neverThemes={['healthcare']}>
+  <Alert variation="error">
+     <p class="ds-c-alert__text">
+        This component is only used for HealthCare. Please use the theme switcher to view the component with HealthCare styles.
+    </p>
+  </Alert>
+</ThemeContent>
+
+## Examples
+
+<StorybookExample
```

Comment:
The old footer page used a responsive example. Should we also provide a responsive example here? I guess they can always click through to storybook and do it there. I do like the minimalism of the normal examples.